client/CSE125Client.vcxproj

```diff
@@ -70,7 +70,7 @@
   </PropertyGroup>
   <ItemDefinitionGroup Condition="'$(Configuration)|$(Platform)'=='Debug|Win32'">
     <ClCompile>
-      <PreprocessorDefinitions>WIN32;_DEBUG;_CONSOLE;%(PreprocessorDefinitions)</PreprocessorDefinitions>

```

Comment:
idk if this should be deleted... maybe just prepend it?
x64 versions didn't have these preprocessor definitions

Reply:
same for line 88